widget/widgetAssets/js/iocExtractionConfigService.service.js

```diff
@@ -62,10 +64,40 @@
             "key",
             "jSONValue"
           ]
+        },
+        globalVariablesToKeyStoreMapping: {
+          "sfsp-excludelist-ips": ["Excludelist_IPs", "IP Address", true],
+          "sfsp-excludelist-urls": ["Excludelist_URLs", "URL", true],
+          "sfsp-excludelist-domains": ["Excludelist_Domains", "Domain", true],
+          "sfsp-excludelist-ports": ["Excludelist_Ports", "Port", true],
+          "sfsp-excludelist-files": ["Excludelist_Files", "File", true],
+          "sfsp-excludelist-cidr-ranges": ["NA", "CIDR Range", true],
+          "sfsp-excludelist-file-hashes":["NA", "File Hash", true],
+          "sfsp-indicator-type-mapping": ["Indicator_Type_Map", "NA", true]
         }
       }
     }
 
+
+    function getIndicatorRegex() {
+      return executeAction('cyops_utilities', 'get_regx_of_indicators', null, []);
```

Comment:
"get_regx_of_indicators" action needs correction, e.g. "get_ioc_validation_regex" "get_ioc_parsing_regex
The should be meaningful.  Also make them const.

Reply:
Sure, will Convey this to Omkar about this